### PR TITLE
Update duux_blizzard_portable_aircon.yaml

### PR DESCRIPTION
**Pull request by nyok92**
update to make it work even better. Tested

### CODE REVIEW

File: custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml [hunk 1]
```diff
@@ -142,3 +152,11 @@ secondary_entities:
             value: Celsius
           - dps_val: true
             value: Fahrenheit
+  - entity: sensor
+    name: countdown timer
```
Comment (make-all) [inline custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml:156]: The first word of entity names should be capitalised according to Home Assistant conventions.  The usual name for this is "Time remaining", or just "Timer" as it is linked to the number Timer above.

**Review by make-all — CHANGES_REQUESTED** [review]

File: custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml [hunk 2]
```diff
@@ -14,14 +14,18 @@ primary_entity:
         - dps_val: true
           constraint: mode
           conditions:
-            - dps_val: cold
+            - dps_val: auto
+              icon: "mdi:hvac"
+              value: auto
+            - dps_val: cool
```
[the remark this answers is not in the document]
Comment (nyok92) [reply custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml:20]: Yes it didn't worked with "cold"

File: custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml [hunk 3]
```diff
@@ -130,7 +140,7 @@ secondary_entities:
         type: boolean
         name: switch
   - entity: select
-    name: Temperature unit
+    name: Temperature display units
```
[the remark this answers is not in the document]
Comment (nyok92) [reply custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml:143]: yep, I will cancel this change

File: custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml [hunk 4]
```diff
@@ -30,14 +34,11 @@ primary_entity:
       name: fan_mode
       mapping:
         - dps_val: low
-          value: low
-        - dps_val: middle
-          value: medium
+          value: Low
```
[the remark this answers is not in the document]
Comment (nyok92) [reply custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml:37]: yep can change back UI names, I just needed to change the dps_value "middle" which was not working for "mid"

File: custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml [hunk 5]
```diff
@@ -113,8 +123,8 @@ secondary_entities:
         name: sensor
         mapping:
           - dps_val: 0
-            value: false
-          - value: true
+            value: False
```
[the remark this answers is not in the document]
Comment (nyok92) [reply custom_components/tuya_local/devices/duux_blizzard_portable_aircon.yaml:126]: sure , we can change it back

**Pull request merged**